src/plugins/intel_gpu/src/kernel_selector/cl_kernels/swiglu_gpu_opt.cl

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// From A&S Handbook of Mathematical Functions with Formulas, Graphs, and Mathematical Tables
+// Formula section 7.1.26
+
+ACCUMULATOR_TYPE fast_erf(ACCUMULATOR_TYPE x) {
```

Comment:
Please use wrappers for functions inside kernels: `FUNC(fast_erf)(ACCUMULATOR_TYPE x)`
With related `FUNC_CALL(fast_erf)(x)` call

Reply:
Acknowledged